src/SprykerEco/Yves/Braintree/Theme/default/components/molecules/braintree-pay-pal/braintree-pay-pal.ts

```diff
@@ -7,83 +7,85 @@ export default class BraintreePayPal extends BraintreePaymentForm {
-            }, function (err, paypalCheckoutInstance) {
+            }, (error, paypalCheckoutInstance) => {
 
-                // Set up PayPal with the checkout.js library
+                // set up PayPal with the checkout.js library
                 paypal.Button.render({
-                    env: self.getAttribute('data-braintree-env'),
-                    locale: self.getAttribute('data-braintree-locale'),
+                    env: this.environment,
+                    locale: this.locale,
                     commit: true,
 
-                    payment: function() {
+                    payment: () => {
                         return paypalCheckoutInstance.createPayment({
                             flow: 'checkout',
                             intent: 'authorize',
-                            amount: self.getAttribute('data-braintree-amount'),
-                            currency: self.getAttribute('data-braintree-currency'),
+                            amount: this.amount,
+                            currency: this.currency,
                             enableShippingAddress: true,
                             shippingAddressEditable: true
                         });
                     },
 
-                    onAuthorize: function (data, actions) {
-                        return paypalCheckoutInstance.tokenizePayment(data).then((payload) => {
-                            payload['amount'] = self.getAttribute('data-braintree-amount');
-                            payload['currency'] = self.getAttribute('data-braintree-currency');
+                    onAuthorize: (data, actions) => {
+                        const nonceInputSelector = <HTMLInputElement>document.querySelector(`input[name='${this.nonceInputName}']`);
 
+                        return paypalCheckoutInstance.tokenizePayment(data).then(payload => {
                             const xhr = new XMLHttpRequest();
                             const userData = JSON.stringify(payload);
 
-                            const nonceInputSelector = <HTMLInputElement>document.querySelector(`input[name='${self.nonceInputName}']`);
+                            payload.amount = this.amount;
+                            payload.currency = this.currency;
                             nonceInputSelector.value = payload.nonce;
-                            self.submitBtn.setAttribute('disabled', 'disabled');
-                            self.form.submit();
+
+                            this.submitBtn.setAttribute('disabled', 'disabled');
+                            this.form.submit();
                         });
                     },
 
-                    onCancel: function (data) {
-                        console.log('checkout.js payment cancelled');
-                        console.log(data);
+                    onCancel: data => {
+                        /* tslint:disable: no-console */
+                        console.log('checkout.js payment cancelled: ', data);
```

Comment:
What is checkout.js?

Reply:
It's a PayPal's checkout.js library, this message comes from official braintree documentation https://developers.braintreepayments.com/guides/paypal/checkout-with-paypal/javascript/v3

Reply:
In any case suggest to add a bit more clear message.